javascript/javascript3/week2/javascript3-homwork-week2/The Happiness Detector/Happy Project/script.js

```diff
@@ -0,0 +1,75 @@
+console.log("Hello");
+document.addEventListener("DOMContentLoaded", function () {
+  // Get the form and submit button
+  const form = document.querySelector("form");
+  const submitBtn = form.querySelector("button[type='submit']");
+
+  // Add a listener for the form submit event
+  form.addEventListener("submit", function (event) {
+    event.preventDefault();
+
+    // Get the input values
+    const name = document.querySelector("#inputName").value;
+    const message = document.querySelector(
+      "textarea[name='contact-message']"
+    ).value;
+    const mood = calculateMood(message);
+
+    if (mood === "happy") {
+      showAlert(
+        `Hey there ${name}. You are clearly happy! :)`,
+        "alert-success"
+      );
+    } else if (mood === "sad") {
+      showAlert(`I'm sorry ${name}. You are clearly sad! :(`, "alert-primary");
+    } else {
+      showAlert(
+        `It's hard to tell how you're feeling, ${name}. :|`,
+        "alert-warning"
+      );
+    }
+  });
+
+  function calculateMood(message) {
```

Comment:
Hahaha. Try to use a sentiment analysis API next time instead of programming your own sentiment analysis.

Reply:
Well, you're doing it now. Maybe not the best algorithm in the world for sentiment analysis but sentiment analysis nonetheless.